app/src/main/res/values/styles.xml

```diff
@@ -62,9 +62,20 @@
         <item name="colorAccent">@color/accent</item>
     </style>
     
-    <!--Theme for Spash-->
+    <!--Theme for Splash-->
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/splash_background</item>
     </style>
 
+    <!--Redesign Login Screen-->
+    <style name="TextInputLayoutLogin" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense">
+        <item name="boxStrokeColor">@color/white</item>
+        <item name="boxStrokeWidth">@dimen/layout_padding_2dp</item>
+    </style>
+
+    <style name="LoginBackground" parent="AppTheme">
```

Comment:
Can we rename it to something better like `LoginTheme`? And add a comment above, like done for `SplashTheme`

Reply:
okay. I'll change it to `LoginTheme`.